tutorials/QML_with_Classiq/QML_with_Classiq.ipynb

```diff
@@ -0,0 +1,1203 @@
+{
```

Comment:
<p>This is nice since our reader might be familiar with Classiq and quantum computing but not PyTorch! :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Thanks :)</p>

Reply:
<p>:)</p>